examples/select-table-rows-by-content/README.md

```diff
@@ -0,0 +1,28 @@
+# Select Table Row by Content
```

Comment:
# Select Table Row by Cell Content

Rename filename, test name and other inclusions too.

Reply:
edited